src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -72,6 +72,14 @@ typedef enum EnInAnimation {
     /* 9 */ ENIN_ANIM_9
 } EnInAnimation;
 
+typedef enum EnInMode {
+    /* 0 */ ENIN_MODE_0,
+    /* 1 */ ENIN_MODE_1,
+    /* 2 */ ENIN_MODE_2,
+    /* 3 */ ENIN_MODE_3,
+    /* 4 */ ENIN_MODE_4
+} EnInMode;
```

Comment:
I would probably make this "init mode" or make it match whatever name you go with.